pkg/app/bootstrap.go

```diff
@@ -42,13 +50,81 @@ var CommandBootstrap = &cli.Command{
 			Usage:   "Dockerhub mirror to use",
 			Aliases: []string{"m"},
 		},
+		&cli.StringSliceFlag{
+			Name:    "ssh-keypair",
+			Usage:   "Manually specify ssh key pair(format: `public_key_path,private_key_path`).",
+			Aliases: []string{"k"},
+		},
 	},
 
 	Action: bootstrap,
 }
 
 func bootstrap(clicontext *cli.Context) error {
+	// If not specified, generate only if key doesn't exist
+	sshKeyPair := clicontext.StringSlice("ssh-keypair")
+	if len(sshKeyPair) != 0 && len(sshKeyPair) != 2 {
+		return errors.Errorf("Invliad ssh-keypair flag")
+	} else if len(sshKeyPair) == 0 {
+
+		keyExists, err := sshconfig.DefaultKeyExists()
```

Comment:
Is it possible, or easy, to show the default key paths in CLI flag?

Reply:
Added